Makefile

```diff
@@ -211,7 +211,7 @@ release-manager-snapshot:
 .PHONY: release-manager-release
 release-manager-release:
 	GO_VERSION=$(shell cat ./.go-version) ./.ci/scripts/install-go.sh
-	$(MAKE) release
+	eval $(shell cat ./go_env.properties) ; $(MAKE) release
```

Comment:
why no `source ./go_env.properties;`

Reply:
Id this failing on our pipelines, if so why we do not use `withGoEnv` or `withMageEnv`

Reply:
no, in the `internal-ci`